src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,35 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[],
+  tempTodo: Todo | null,
+  deleteTodo: (id: number) => void,
+  processedIds: number[],
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  deleteTodo,
+  processedIds,
+}) => (
+  <ul style={{ listStyleType: 'none' }}>
```

Comment:
Consider using CSS classes or styled-components instead of inline styles for better maintainability.